notebooks/ACS/acs_focus_diverse_epsfs/acs_focus_diverse_epsfs.ipynb

```diff
@@ -0,0 +1,422 @@
+{
```

Comment:
<p>Hi @gsanand, running this notebook locally works well for file location as you mentioned, but it seems the CI environment is having trouble locating the file. To improve this, I have a couple of suggestions. First, consider specifying a specific download directory instead of using './'. This can help ensure consistent file paths in the CI environment, where the directory structure often differs from local setups. Second, using os.path.abspath to convert the relative path to an absolute one could clarify the file's location, aiding the CI system in locating the file more reliably. Additionally, I recommend using a context manager when opening files with fits.open.&nbsp;</p><p> </p><pre class="ql-syntax" spellcheck="false">import os&nbsp;# add this with other imports
# Set the download location to the current working directory
download_location = os.path.join(os.getcwd(), 'downloads')
os.makedirs(download_location, exist_ok=True)
retrieved_download = psf_retriever('jds408jsq', download_location)&nbsp;
retrieved_filepath = os.path.abspath(retrieved_download)
if not os.path.isfile(retrieved_filepath):
&nbsp;&nbsp;raise FileNotFoundError(f"Expected file not found at {retrieved_filepath}")
with fits.open(retrieved_filepath) as hdu:
&nbsp;&nbsp;hdu.info() # Display basic information about the file
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
hi @haticekaratay, I tried your suggestions above, and while it still works locally, it does not get rid of the CI errors on git. any thoughts as to what is going on? the errors are sufficiently vague enough that I'm not entirely sure what the issue is

Reply:
Thanks for the update, @mgough-970; I will handle the branching issue.